docs/dev/release.md

```diff
@@ -73,6 +73,12 @@ git remote add upstream https://github.com/redhat-openshift-ecosystem/certified-
 export RH_CERTIFIED_OPENSHIFT_REPO_PATH=$PWD/certified-operators
 ```
 
+### Before creating PRs against RedHat repositories
```

Comment:
This should be in a `### Prerequisites` section at the beginning of the document.

Reply:
Moved